### PR TITLE
update package.json and README for ui dev set up

### PR DESCRIPTION
**Pull request by koesbong**
This sets a fixed node.js version to 12, as well as updating the README for setting up a local dev environment.

### CODE REVIEW

File: ui/README.md [hunk 1]
```diff
@@ -7,16 +7,17 @@ A short introduction of this app could easily go here.
...
 You will need the following things properly installed on your computer.
 
-- [Git](https://git-scm.com/)
-- [Node.js](https://nodejs.org/) (with npm)
+- [Node.js v12](https://nodejs.org/)
+  - The current codebase has been tested to run well with `node` version 12 so it is _**strongly recommended**_ that you use this version
+  - You can use node version managers to manage all of your node versions, for example [nvm](https://github.com/nvm-sh/nvm), [n](https://github.com/tj/n), etc.
+- [Yarn](https://classic.yarnpkg.com/en/docs/install)
 - [Ember CLI](https://ember-cli.com/)
-- [Google Chrome](https://google.com/chrome/)
```
Comment (krantzinator) [inline ui/README.md:13]: Chrome is required for some of the tests I believe.

**Review by krantzinator — CHANGES_REQUESTED** [review]

File: ui/README.md (hunk 1 above)
Comment (koesbong) [reply ui/README.md:13]: Added back

**Review by krantzinator — APPROVED** [review]
thanks for cleaning this up!

**Review by gregone — APPROVED** [review]
Great! Thanks for updating this, @koesbong !

**Pull request merged**